src/taco-cli/test/templateManager.ts

```diff
@@ -151,82 +151,25 @@ describe("TemplateManager", function (): void {
-                    // Verify the returned path is correct
-                    cachedTemplatePath.should.equal(cachedTestTemplate);
-
-                    // Verify the TemplateManager didn't re-extract the template to the cache: the cached template should only contain the single file we placed there
-                    // at the start of this test, "a.txt", otherwise it means the TemplateManager re-extracted the template even though it was already cached
-                    fs.existsSync(path.join(cachedTestTemplate, "folder1")).should.be.exactly(false, "The template archive should not have been re-extracted to the cache");
-                    done();
-                })
-                .catch(function (err: string): void {
-                    done(new Error(err));
-                });
-        });
-
-        it("should correctly find and cache a template that is not already cached", function (done: MochaDone): void {
-            // Create a test TemplateManager
-            var templates: templateManager = new templateManager(mockKitHelper, templateCache);
+            // Call acquireFromTacoKits()
+            (<any> templates).acquireFromTacoKits(testTemplateId, testKitId, runFolder)
+                .then(function (tempTemplatePath: string): void {
+                    // Verify the returned path is under the right folder
+                    tempTemplatePath.indexOf(runFolder).should.equal(0);
 
-            // Make sure the template cache is empty
-            fs.existsSync(templateCache).should.equal(false, "Test template cache must be initially empty for this test");
+                    // Verify the parent directory starts with the proper prefix
+                    var parentDirectoryName: string = path.basename(path.dirname(tempTemplatePath));
 
-            // Call findTemplatePath()
-            (<any> templates).findTemplatePath(testTemplateId, testKitId, testTemplateOverrideInfo.templateInfo)
-                .then(function (cachedTemplatePath: string): void {
-                    // Verify the returned path is correct
-                    cachedTemplatePath.should.equal(cachedTestTemplate);
+                    parentDirectoryName.indexOf("taco_template_").should.be.equal(0);
 
-                    // Verify the TemplateManager correctly extracted the template archive to the cache
-                    fs.existsSync(path.join(cachedTestTemplate, "a.txt")).should.be.exactly(true, "The template was not correctly cached (missing some files)");
-                    fs.existsSync(path.join(cachedTestTemplate, "folder1", "b.txt")).should.be.exactly(true, "The template was not correctly cached (missing some files)");
-                    done();
-                })
-                .catch(function (err: string): void {
-                    done(new Error(err));
-                });
-        });
-
-        it("should return the appropriate error when templates are not available", function (done: MochaDone): void {
```

Comment:
This test can be retained for testing the negative cases - i.e, cases when TacoErrorCodes.CommandCreateTemplatesUnavailable is returned.